public/coffee/ide/share/controllers/ShareProjectModalController.coffee

```diff
@@ -60,36 +70,41 @@ define [
 				$scope.inputs.contacts = []
 				$scope.state.error = null
 				$scope.state.inflight = true
-				
+
 				currentMemberEmails = getCurrentMemberEmails()
 				do addNextMember = () ->
 					if members.length == 0 or !$scope.canAddCollaborators
 						$scope.state.inflight = false
 						$scope.$apply()
 						return
-					
+
 					member = members.shift()
 					if !member.type? and member.display in currentMemberEmails
 						# Skip this existing member
 						return addNextMember()
-					
+
+					# NOTE: groups aren't really a thing in ShareLaTeX, partially inherited from DJ
 					if member.type == "user"
-						request = projectMembers.addMember(member.email, $scope.inputs.privileges)
+						request = projectInvites.sendInvite(member.email, $scope.inputs.privileges)
 					else if member.type == "group"
 						request = projectMembers.addGroup(member.id, $scope.inputs.privileges)
 					else # Not an auto-complete object, so email == display
-						request = projectMembers.addMember(member.display, $scope.inputs.privileges)
```

Comment:
[this bit of code is gone now, but for some reason it's still showing up in the github comments UI]